TalkAloud/TalkAloud/AudioViewController.swift

```diff
@@ -9,16 +9,16 @@
 import UIKit
 import AVFoundation
 
-class AudioViewController: UIViewController {
-    
+class AudioViewController: UIViewController, AudioEngineStateChangeDelegate {
     // Intialized AudioEngine object so properties and methods can be used for later
     var audioEngine = AudioEngine()
-    
+    var delegate: AudioEngineStateChangeDelegate?
```

Comment:
Why this here?

Reply:
Good question. I didn't need it there just made a mistake when I was implementing the delegate method and I thought I needed it.